cli/src/commands/util.rs

```diff
@@ -230,6 +271,24 @@ fn cmd_util_config_schema(
     Ok(())
 }
 
+fn cmd_util_exec(args: &UtilExecArgs) -> Result<(), CommandError> {
+    let status = std::process::Command::new(&args.command)
+        .args(&args.args)
+        .spawn()
+        .map_err(|e| CommandError::new(CommandErrorKind::User, e))?
+        .wait()
+        .unwrap();
+    if !status.success() {
+        let error_msg = if let Some(code) = status.code() {
+            format!("external command failed with exit code {code}")
+        } else {
+            "external command was terminated by signal".into()
+        };
```

Comment:
nit: `ExitStatus` implements `Display`. Something like this should work: `"External command exited with {exit_status}"`